ribs/emitters/rankers.py

```diff
@@ -121,10 +123,13 @@ class TwoStageImprovementRanker(RankerBase):
 
     def rank(self, emitter, archive, rng, solution_batch, objective_batch,
              measures_batch, metadata, add_statuses, add_values):
+        # To avoid using an array of tuples, ranking_values is an 2D array
+        # [[value_0, status_0], ..., [value_n, status_n]]
+        ranking_values = np.stack((add_values, add_statuses), axis=-1)
```

Comment:
I think it makes more sense for the statuses to come first in the output since that conforms with the way Python does things. You can include a note about how lexsort requires reversing the order.